interface/forms/soap/templates/general_new.html

```diff
@@ -1,52 +1,58 @@
-                        <div class="col-sm-10">
-                            <textarea name="plan" id="plan" class="form-control" cols="60" rows="6">{$data->get_plan()|escape:'html'}</textarea>
-                        </div>
-                    </div>
-                    <div class="form-group">
-                        <div class="col-sm-offset-2 col-sm-10">
-                            <div class="btn-group" role="group">
+                <form name="soap" method="post" action="{$FORM_ACTION}/interface/forms/soap/save.php" onsubmit="return top.restoreSession()">
+                    <fieldset>
+                            <legend class="">{xl t='Subjective'|escape:'html'}</legend>
+                                <div class="form-group" >
+                                    <div class="col-sm-10 col-sm-offset-1">
+                                        <textarea name="subjective" class="form-control" cols="40" rows="6">{$data->get_subjective()|escape:'html'}</textarea>
+                                    </div>
+                                </div>
+                    </fieldset>
+                    <fieldset>
+                            <legend class="">{xl t='Objective'|escape:'html'}</legend>
+                                <div class="form-group">
+                                    <div class="col-sm-10 col-sm-offset-1">
+                                        <textarea name="objective" class="form-control" cols="60" rows="6">{$data->get_objective()|escape:'html'}</textarea>
+                                    </div>
+                                </div>
+                    </fieldset>
+                    <fieldset>
+                            <legend class="">{xl t='Assessment'|escape:'html'}</legend>
+                                <div class="form-group">
+                                    <div class="col-sm-10 col-sm-offset-1">
+                                        <textarea name="assessment" class="form-control" cols="60" rows="6">{$data->get_assessment()|escape:'html'}</textarea>
+                                    </div>
+                                </div>
+                    </fieldset>
+                    <fieldset>
+                            <legend class="">{xl t='Plan'|escape:'html'}</legend>
+                                <div class="form-group">
+                                    <div class="col-sm-10 col-sm-offset-1">
+                                        <textarea name="plan" class="form-control" cols="60" rows="6">{$data->get_plan()|escape:'html'}</textarea>
+                                    </div>
+                                </div>
+                    </fieldset>
+                    <?php //can change position of buttons by creating a class 'position-override' and adding rule text-align:center or right as the case may be in individual stylesheets ?>
+                    <div class="form-group clearfix">
```

Comment:
I have seen responsiveness lost in other forms where elements are not confined to their divs